### PR TITLE
314 docfx code highlight and edits

### PR DESCRIPTION
**Pull request by StarfallProjects**
This was originally for Issue 314 (highlight code) It escalated. I have:

- Added highlights in one or two places. Note that as you can only highlight full lines, not individual terms/words, this is probably not the result originally intended.
- Fixed headings to sentence case
- Spotted a few inconsistencies. Fixed where I can. I have a list of things I need further info/decisions on in a google doc [here](https://docs.google.com/document/d/1yi4rWHZJPe3sDgC_rBWmBgKTgX0yuQnutO4CF-AMO0Q/edit?usp=sharing) This PR will require more work based on your responses.
- Other minor edits

### CODE REVIEW

File: dotnet-api/stream-metadata.md [hunk 1]
```diff
@@ -2,39 +2,13 @@
...
 ---
 
-# Stream Metadata
+# Stream metadata
 
 Every stream in Event Store has metadata associated with it. Internally, the metadata includes information such as the ACL of the stream and the maximum count and age for the events in the stream. Client code can also put information into stream metadata for use with projections or through the client API.
 
 This information is not part of the actual event but is metadata associated with the event. Event Store stores stream metadata as JSON, and you can access it over the HTTP APIs.
 
-## Methods
```
Comment (ChrisChinchilla) [inline dotnet-api/stream-metadata.md]: @StarfallProjects I appreciate this doesn't add that much, but why remove it?

Comment (ChrisChinchilla) [reply dotnet-api/stream-metadata.md]: At the moment it's duplicating the autogenerated content anyway and I want to think of a clever way to reduce, but don't have one right now.

File: server/cluster-with-manager-nodes.md [hunk 2]
```diff
@@ -16,18 +16,18 @@ This document covers setting up Event Store with manager nodes and database node
...
 [!include[<Node number>](~/partials/_cluster-size.md)]
 
-## Manager Nodes
+## Manager nodes
 
 Each physical (or virtual) machine in an Event Store cluster typically runs one manager node and one database node. It's also possible to have multiple database nodes per physical machine, running under one manager node.
 
 Manager nodes have a number of responsibilities:
 
--   They are responsible for starting the database nodes and supervising them to ensure they rest in case of a crash or termination owing to abnormal circumstances. Event Store calls this the _watchdog_ service.
```
Comment (ChrisChinchilla) [inline server/cluster-with-manager-nodes.md]: @StarfallProjects I know we haven't made that style guide yet, but personally I'd like to have terms like this stand out somehow, quotes, italics, bold, don't really mind.

File: server/scavenging.md [hunk 3]
```diff
@@ -27,15 +27,16 @@ You start a scavenge by issuing an empty `POST` request to the HTTP API with the
...
-> [!NEXT]
-> Scavenge operations have other options you can set to improve performance, [read the API docs](#eventstore_org_HTTP_API_5_0_0_Scavenge_a_node) for more details.
-
 ### [Response](#tab/tabid-9)
 
 [!code-http[server-scavenge-response](~/code-examples/server/scavenge.sh?range=3-)]
 
 * * *
 
+<br>
```
Comment (ChrisChinchilla) [inline server/scavenging.md]: @StarfallProjects Shouldn't need any HTML

File: dotnet-api/stream-metadata.md (hunk 1 above)
Comment (StarfallProjects) [reply dotnet-api/stream-metadata.md]: Heading levels. It was either remove that, or change the headings below to H3. I tried it that way, and they didn't look right - the headings below are important enough they should be h2 (think it also affected whether they showed on the menu)

File: server/cluster-with-manager-nodes.md (hunk 2 above)
Comment (StarfallProjects) [reply server/cluster-with-manager-nodes.md]: Reverted

File: server/scavenging.md (hunk 3 above)
Comment (StarfallProjects) [reply server/scavenging.md]: Agreed - was trying to stop the code and tips joining up. Can I try and fix it in the theme CSS? Assume adding a margin to code snippets would be the way forward?

File: dotnet-api/stream-metadata.md (hunk 1 above)
Comment (ChrisChinchilla) [reply dotnet-api/stream-metadata.md]: But unless I am confused, it also removes the code?

File: server/scavenging.md (hunk 3 above)
Comment (ChrisChinchilla) [reply server/scavenging.md]: Ah, let me take a look when I merge this anyway

**Pull request merged**